### PR TITLE
The Long-Awaited HoP Hardsuit

### PR DESCRIPTION
**Pull request by RaveRadbury**
hardsuithop

## About The Pull Request

Adds the Head of Personnel's Hardsuit and adds a HoP suit storage to each map.

## Why It's Good For The Game

All the other heads have hard suits, now the HoP does too, sort of.

This is an attempt to play into the rivalry/power-dynamic between HoP and Cap, giving HoP an inferior Cap hardsuit.

## Changelog
:cl: Raveradbury, Navi.os
add: Added hardsuit for Head of Personnel
add: Added suit storage units for Head of Personnel
imageadd: HoP Hardsuit + icon
imageadd: HoP Hardsuit Helmet + icon
/:cl:

Someone is working on the art for this. Should be up later tonight.

Also, totally open to altering the balance on this. Maybe remove the heat and fire resistance, too? Or knock all of the stats down a bit? There are many options.

Feeling cute, might close later, idk

TODO
 - [x] Exists
 - [x] Spawns on map
 - [x] Basic Visuals
 - [x] Better Visuals
 - [ ] Holds pet carrier on back slot
 - [ ] Balanced Armor
 - [ ] Balanced Spaceworthiness
 - [ ] Balanced in general

### CODE REVIEW

**Comment by Shapsy** [comment]
putting aside the very obvious concerns about giving the HoP, who already has access to EVA and Cargo, a free copy of the best hardsuit in the game


why did you decide to mark it as okay for showers?

**Comment by RaveRadbury** [comment]
Oddly enough all of the hardsuits are SHOWEROKAY. I'm just following standards.

You're right about the stats being too much. I'll knock em down.

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -573,6 +574,23 @@
...
+	item_state = "hopspacehelmet"
+
+
+/obj/item/clothing/suit/space/hardsuit/swat/hop
+	name = "HoP's SWAT suit"
+	desc = "A MK.II SWAT suit with streamlined joints and armor made out of superior materials, insulated against intense heat. The most advanced tactical armor available. Usually reserved for heavy hitter corporate security, this one has a regal finish in Nanotrasen company colors. Better not let the assistants get a hold of it. Something about it seems a bit off."
+	clothing_flags = THICKMATERIAL | SHOWEROKAY
+	icon_state = "hoparmor"
+	item_state = "hopspacesuit"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/swat/captain
```
Comment (GunHog) [inline code/modules/clothing/spacesuits/hardsuit.dm]: This should probably be `/obj/item/clothing/head/helmet/space/hardsuit/swat/hop`

**Comment by GunHog** [comment]
You missed a golden opportunity to make a hardsuit modeled after Ian.

**Comment by RaveRadbury** [comment]
> You missed a golden opportunity to make a hardsuit modeled after Ian.

There's still time. There's an Ian costume in the files and it would just be a matter of using that.

It's just so incredibly silly, they'd basically be like pajamas.

**Comment by GunHog** [comment]
Well, I am not thinking "LOL DOGSUIT", but to have the trim in be Ian's colors or something. Imagine if you, in a corporate or military setting, had a choice in the color of your armor. You would probably trim it in the colors of what you love most. It would not look crazy or out of place like a dogsuit, it would just be a small nod to Ian.

**Comment by RaveRadbury** [comment]
I'm in touch with someone who actually arts, I'll pass this along

**Comment by RaveRadbury** [comment]
Having the Ian connection will probably cause HoP's to choose it over all others. Pretty smart.

**Comment by HomocidlePancakes** [comment]
>why did you decide to mark it as okay for showers?

Because all spesssuits have that? Duh

**Comment by HomocidlePancakes** [comment]
Have an internal pet carrier or something if you won’t give Ian a separate spessuit

**Comment by bgobandit** [comment]
why would the HoP have or need a hardsuit when his job is pushing paper

**Comment by RaveRadbury** [comment]
> why would the HoP have or need a hardsuit when his job is pushing paper

This is why I was thinking of having it not be space-worthy.

Head hardsuits aren't just space suits, they are also suits of armor. The HoP is an arguably larger target than the other heads. I get that his benefits are supposed to balance for his lack of hardsuit, yet I also see a lot of HoP's taking the Captain's hardsuit.

If we pull this off right, the HoP will have a hardsuit that has a minor amount of defense to justify the irresistible call of the HoP's bond with Ian. Then we have players operating with IC clothing that maintains the intended strengths and weaknesses of the character.

Also, why does the CMO have a hard suit, really. There was a lot of discussion in the CMO hardsuit buff PR recently and it kind of highlighted how situational it was.

**Comment by RaveRadbury** [comment]
> Have an internal pet carrier or something if you won’t give Ian a separate spessuit

I bet I can make it hold pet carrier in back slot.

**Comment by anconfuzedrock** [comment]
The main thing is that aside from having eva access and not fucking needing a hardsuit, the HoP could use a mining hardsuit at minimum. yet another reason lavaland is gay god I want to pee on kor

**Comment by necromanceranne** [comment]
The intention of the CMO's hardsuit is that it was dual-purpose rad/bio protection and corpse retrieval from space. It rarely gets used because frankly it is silly that it is exclusive to the CMO. It would be significantly more useful if medical doctors could access them.

Now it's funny that the mining hardsuit should come up since honestly that's a pretty decent comparison to what I think a HoP hardsuit should be. Decent environment protections, fuck awful hazard protections, hefty melee protections. It's the dirty greyshirts who are the HoP's true enemy.

**Comment by habader** [comment]
give it negative armour against everything

**Comment by AzureWildfire** [comment]
I'm not sure about this, but you better not give it a headlight, because the cap suit doesn't have one, it cannot be better than the caps one. 
Also i'm up to a miner hardsuit for the HoP, it would make more sense than a caps suit for the HoP. then he can have a suit headlight. (Of course with different design than the miner hardsuit)

**Comment by anconfuzedrock** [comment]
Also this is disgustingly ugly

**Comment by knacker48** [comment]
Whats with the mouth? Why is it yellow?

**Comment by habader** [comment]
> Whats with the mouth? Why is it yellow?
cheetos dust

**Comment by ExcessiveUseOfCobblestone** [comment]
If anything the hop hardsuit should double as a photo-copier.

**Comment by necromanceranne** [comment]
In-built PDA so you can hands-free tell cargo to order another corgi crate.

**Comment by PKPenguin321** [comment]
I remember when this was a sarcastic, cynical joke when they were adding the RD's hardsuit

**Comment by tralezab** [comment]
The real power move would be removing every hardsuit (except the Chief Engineer's) until the department researches it, and the printers can only make one for each department.

...But oh well, we're heading deeper into the station having every thing they could possibly want roundstart. And we're doing it with that THING y'all calling a sprite

**Comment by dr-bee** [comment]
Why is the HOP hardsuit not just a literal business suit that happens to be over-starched?

**Comment by WJohn** [comment]
Yeah that's a really bad sprite that is very clearly just a *very light* reskin from the captain's hardsuit.

HoP doesn't need his own space suit his office is always right next to like, 8 of them.

**Comment by optimumtact** [comment]
I can't believe we have gotten to the level of unironically doing this

**Comment by RaveRadbury** [comment]
I can't believe so many people missed that the hardsuit had its spaceworthiness removed.

**Comment by RaveRadbury** [comment]
> Whats with the mouth? Why is it yellow?

It's a beard. The appearance settings were random.

**Comment by Farquaar** [comment]
I don't really see the point of having a hardsuit without space protection. I always figured that the officer hardsuits were so the heads had a better shot at survival in an emergency situation (i.e. massive hull breach, not just for flashy armor.

I'm in favour of a HoP hardsuit, but maybe not with these stats and this sprite.

**Comment by Barhandar** [comment]
>hardsuit that is not spaceworthy

Why call it a hardsuit at all then? There's already precedent for non-hardsuits with collapsible helmets, and _all_ other hardsuits are spaceproof.

**Comment by ExcessiveUseOfCobblestone** [comment]
when are we getting a company employee mech that has a built in paper bin, mech-powered pen, and copier that only breaks every other day?

**Pull request closed without merging**